core/src/main/java/com/linecorp/armeria/common/multipart/DefaultMultipart.java

```diff
@@ -40,12 +48,16 @@
 import io.netty.buffer.ByteBufAllocator;
 import io.netty.util.concurrent.EventExecutor;
 
-final class DefaultMultipart implements Multipart {
+final class DefaultMultipart implements Multipart, StreamMessage<HttpData> {
 
     /**
-     * The default boundary used for encoding multipart messages.
+     * Returns a random boundary used for encoding multipart messages.
      */
-    static final String DEFAULT_BOUNDARY = "!@==boundary==@!";
+    static String randomBoundary() {
+        final byte[] bytes = new byte[16];
+        ThreadLocalRandom.current().nextBytes(bytes);
+        return Base64.getEncoder().encodeToString(bytes);
```

Comment:
- This will add a padding like `==`. Can we use `BaseEncoding` in Guava to customize it a little bit? i.e. `private static final BaseEncoding base64 = BaseEncoding.base64().omitPadding()`
- Should we prepend a static string like `ArmeriaBoundary`, just like Webkit does? https://stackoverflow.com/a/42985029/55808